src/eureka/S3_data_reduction/background.py

```diff
@@ -152,7 +152,21 @@ def writeBG_WFC3(arg):
     data['flux'] -= data.bg
     if hasattr(data, 'medflux'):
         data['medflux'] -= np.median(data.bg, axis=0)
-
+        
+    # Save BG value at source position and BG stddev (no outlier rejection)
+    data['skylev'] = (['time', 'x'], 
+                      np.zeros([data.flux.shape[0], data.flux.shape[2]]))
+    data['skylev'].attrs['flux_units'] = data['flux'].attrs['flux_units']
+    data['skylev'] = data.bg[:, meta.src_ypos, :]
+    
+    data['skyerr'] = (['time', 'x'], 
+                      np.zeros([data.flux.shape[0], data.flux.shape[2]]))
+    data['skyerr'].attrs['flux_units'] = data['flux'].attrs['flux_units']
+    y_inds = np.zeros([data.flux.shape[1]], dtype=bool)
+    y_inds[:meta.bg_y1] = True
+    y_inds[meta.bg_y2:] = True
+    data['skyerr'] = np.std(data.flux[:, y_inds, :], axis=1)
```

Comment:
I think we should start with data.mask here, instead of all zeros.  Something like `y_inds = deepcopy(data.mask)`.  Since the masks in Eureka! now set bad values to True, we would set `y_inds[:, meta.bg_y1:meta.bg_y2] = True`.  

The std dev calculation may also require an update based on the new mask implementation, but I'm not certain.  Maybe it would be easiest to create a masked array and then do the calculation?

Reply:
I thought about this and came to the conclusion that I didn't want to mask bad data before the stddev here because then larger a stddev would indicate there were bad pixels in the column, resulting in fewer pixels for measuring background, resulting in maybe less accurate background removal. Happy to discuss this though.

Reply:
Rather than using just the standard deviation, I'd think instead using the standard error of the mean (aka `np.ma.std(values)/np.sqrt(np.sum(~values.mask))`) would strike a good balance. With the current setup, users might be concerned about seemingly noisy background measurements even though the bad pixels driving that noise were actually masked. And by dividing by the square-root of the number of unmasked points, you still capture the increased uncertainty in columns where the background measurement was likely a bit noisier since it was based on fewer measurements